.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10, 12, 14, 16]
+        node-version: [12, 14, 16]
```

Comment:
Is it a must to drop `node@10`? If yes, I think we can simplify the process and target to a major bump.

Reply:
This is going to be a major bump anyway

Reply:
It's a must - with a new release of `fast-jwt` we dropped support for v10 as it uses some functionalities from the newer versions. As it is mentioned earlier it is going to be a major version.